docs/source/helpers.rst

```diff
@@ -4,8 +4,9 @@ Helpers
 
 Helpers are higher level functions built for reducing recurring code boilerplate.
 We add helpers as method of ``CSTNode`` or ``libcst.helpers`` package based on those principles:
-  - ``CSTNode`` method: simple, read-only and only require data of the direct children of a CSTNode.
-  - ``libcst.helpers``: node transforms or require recursively traversing the syntax tree.
+
+- ``CSTNode`` method: simple, read-only and only require data of the direct children of a CSTNode.
+- ``libcst.helpers``: node transforms or require recursively traversing the syntax tree.
```

Comment:
Previously the indent makes the bullet points indented because it's part of the prior paragraph.
![image](https://user-images.githubusercontent.com/3840867/72636682-70a1c300-3914-11ea-900d-6f38498d4a27.png)

This removes the indent. Is this better?

Reply:
It removes the warning about this section in the docs build. I don't particularly care how its formatted, but the builds need to stay clean.

Reply:
Where do you see the warning. I see nothing when running `tox -e docs`.

Reply:
![image](https://user-images.githubusercontent.com/421365/72637538-6aace180-3916-11ea-8791-b3980fefb1fd.png)

Reply:
I figure it out that it can be fixed by adding a blank line before the indentation.
Do you mind just add one blank line before `- ``CSTNode`` method: ...` instead of removing indentation?

Reply:
No, I'm not doing your work for you. You can put up a PR to fix it if it bothers you.